Lodgify.Payments.Stripe.Server.IntegrationTests/Factories/CustomWebApplicationFactory.cs

```diff
@@ -19,7 +19,8 @@
 
 namespace Lodgify.Payments.Stripe.Server.IntegrationTests.Factories;
 
-public class CustomWebApplicationFactory : WebApplicationFactory<Program>, IAsyncLifetime
+public class CustomWebApplicationFactory<T> : WebApplicationFactory<Program>, IAsyncLifetime
```

Comment:
I am not a huge fun of `Custom` word how about:
```suggestion
public class TestWebApplicationFactory<T> : WebApplicationFactory<Program>, IAsyncLifetime
```

Reply:
I was follow with that "convention" base on Microsoft documentation, and habbits that everybody call this class in that way, but no problem ;)